portfolio/src/main/webapp/templates/comments.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Comment Section</title>
+    <link rel="stylesheet" href="../styles/comments.css">
+  </head>
+  <body onload="postComment()">
+	<div id="comment-section">
+    <div id="input-area>">
+    	<h2>Comment section</h2>
+    	<form id="form" autocomplete="off" action="/create-new-comment" method="POST">
+				<label for="username">Name:</label>
+				<input type="text-input" name="username">
+				<label for="comment">Leave a comment here:</label>
+				<input type="text-input" name="comment">
+				<input type="submit" value="Submit">
+    	</form> 
+		</div>
+		<div id="output-area">
+      <label for="number-comments">Maximum number of comments to display:</label> 
+			<input type="number" name="number-comments" id="number-comments" min="1" max="50" value="1">
+      <ul id="comment-display"></ul>
```

Comment:
It's a \<ul\> but you're populating it with divs, not \<li\> elements.  So just make it a div.